content/en/boilerplates/snips/gateway-api-install-crds.sh

```diff
@@ -22,5 +22,5 @@
 
 bpsnip_gateway_api_install_crds_install_crds() {
 kubectl get crd gateways.gateway.networking.k8s.io &> /dev/null || \
-  { kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd?ref=v1.1.0" | kubectl apply -f -; }
+  { kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.1.0/standard-install.yaml; }
```

Comment:
{{< k8s_gateway_api_version >}} again

Reply:
Yeah this is autogen'd from the `.md` snip - it looks like the hard version is expanded+stamped out in the script even if you use the placeholder in the `md`.